lib/core/cne/meson.build

```diff
@@ -4,6 +4,7 @@
 sources = files(
     'cne.c',
     'uid.c',
+    'tailqs.c',
```

Comment:
super minor nit, should we keep this alphabetized?

Reply:
I pushed the change to reorder the lines and that was the only change. I will merge after the tests have finished.